pkg/identityserver/application_access.go

```diff
@@ -227,6 +228,8 @@ func (is *IdentityServer) getApplicationCollaborator(ctx context.Context, req *t
 	return res, nil
 }
 
+var errApplicationNeedsCollaborator = errors.DefineFailedPrecondition("application_needs_collaborator", "every application needs at least one collaborator with all rights")
```

Comment:
Can't we use one message with an attribute for the entity type for all messages?

Reply:
Sure, but then we'll have a random entity type variable in the message that we can't translate.